examples/forward/point_mass.py

```diff
@@ -0,0 +1,53 @@
+"""
+Point Mass Forward Modelling on Cartesian Coordinates
+=====================================================
+
+The simplest geometry used to compute gravitational fields are point masses.
+Although modelling geologic structures with point masses can be challenging, thery are
+very usefull for other purposes: creating synthetic models, solving inverse problems
+very quickly, generating equivalent sources for interpolation or gridding, etc.
+Besides, the gravitational fields generated by point masses can be quickly computed
+either in Cartesian or in geocentric spherical coordinate system.
+We will compute the downward component of the gravitational acceleration generated by
+a set of point masses on a computation grid given in Cartesian coordinates. We will do
+it throught the :func:`harmonica.point_mass_gravity` function.
+"""
+import harmonica as hm
+import verde as vd
+import matplotlib.pyplot as plt
+import matplotlib.ticker
+
+
+# Define two point masses with oposite mass values of 10000000 kg
+easting = [5e3, 15e3]
+northing = [5e3, 15e3]
+vertical = [-5e3, -2.5e3]
```

Comment:
Wasn't z pointing downward? If so, the masses should be at positive z and computation grid at negative z, right?

Reply:
The `vertical` coordinates are assumed to be positive upwards, that's why the depths of the point masses are negatives. But the `g_z` is assumed to be positive downwards.
Now I think about it, it's very confusing. I think we should standardize it to z -> down. What bothers me is the name `vertical`, I don't know why, but my head pictures it as an arrow pointing upwards. Possible alternatives could be `depth` or `down`. What do you think?

Reply:
I changed the direction of z to downwards, but left the variables named as `vertical` until we agree a best name.

Reply:
I was thinking a little bit about this, and I like the `down` alternative. So I renamed all `vertical` appearances with `down` and added a warning when working with Cartesian coordinates. Let me know if you don't like it.

Reply:
`down` sounds good to me :+1: